### PR TITLE
Update sample: analyze hotspots

### PR DESCRIPTION
**Pull request by ruiqima**
## Description

PR to update the sample to use composable mapview from the toolkit.

## Links and Data

Sample Epic: runtime/kotlin/issues/#3364

## What To Review

- Review updated files

## How to Test

Run the sample on the sample viewer or the repo.

### CODE REVIEW

**Comment by ruiqima** [comment]
One question about the `onSingleTapConfirmed = {}` param in `MapView - MainScreen`. 

Previously, we have a default value `{}` defined for `ComposableMapView` as `fun ComposeMapView(/*...*/, onSingleTap: (SingleTapConfirmedEvent) -> Unit = {}`, but no value is set for `onSingleTap` when calling `ComposeMapView`. So it actually does nothing.

After migrating to use composable MapView, should I intently keep it here as `MapView(/*..*/, onSingleTapConfirmed = {})`, or should I remove this?

File: analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt [hunk 1]
```diff
@@ -36,21 +41,31 @@ import kotlinx.coroutines.launch
...
-                ComposeMapView(
-                    modifier = Modifier.fillMaxSize().weight(1f),
-                    mapViewModel = mapViewModel
+                MapView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
+                    arcGISMap = mapViewModel.map,
+                    viewpointOperation = MapViewpointOperation.Set(viewpoint = viewpoint),
+                    onSingleTapConfirmed = {}
```
Comment (ruiqima) [inline analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt:68]: Previously, we have a default value defined for ComposableMapView as fun ComposeMapView(/*...*/, onSingleTap: (SingleTapConfirmedEvent) -> Unit = {}), but no value is set for onSingleTap when calling ComposeMapView. So it actually does nothing.

After migrating to use composable MapView, should I intentionally keep it here as MapView(/*..*/, onSingleTapConfirmed = {}), or should I remove this?

File: version.gradle [hunk 2]
```diff
@@ -2,7 +2,7 @@ ext {
     // ArcGIS Maps SDK for Kotlin version
     arcgisVersion = '200.4.0-4085'
     // ArcGIS Maps SDK for Kotlin Toolkit version
-    arcgisToolkitVersion = '200.3.0-1'
```
Comment (prupani-7) [inline version.gradle:5]: this change is not needed, if you merge with latest feature branch, it should update

**Review by prupani-7 — CHANGES_REQUESTED** [review]
@ruiqima Sample changes look good 👍 
added some comments

File: analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt [hunk 3]
```diff
@@ -36,21 +41,30 @@ import kotlinx.coroutines.launch
...
-                // composable function that wraps the MapView
-                ComposeMapView(
-                    modifier = Modifier.fillMaxSize().weight(1f),
-                    mapViewModel = mapViewModel
+                MapView(
+                    modifier = Modifier
+                        .fillMaxSize()
+                        .weight(1f),
+                    arcGISMap = mapViewModel.map,
+                    viewpointOperation = MapViewpointOperation.Set(viewpoint = viewpoint)
```
Comment (prupani-7) [inline analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt:67]: could we rename the `viewpoint` variable to something else for code clarity or we could even directly substitute the Viewpoint() here without having to declare it above.
We did the same in previous samples.

**Review by prupani-7 — APPROVED** [review]
Thanks for making the changes, looks good! 
I have added a comment on `viewpoint` var. It would be better to rename the var or to directly substitute it with Viewpoint() object declared above.

But happy to approve this. 👍

File: analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt [hunk 4]
```diff
@@ -36,21 +41,30 @@ import kotlinx.coroutines.launch
...
                 )
                 // bottom layout with a button to display analyze hotspot options
                 BottomAppContent(
```
Comment (changanxian) [inline analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt:70]: at line #100, change `.value` to `.intValue`

**Review by changanxian — APPROVED** [review]

**Comment by changanxian** [comment]
@ruiqima looks good! 👍

File: analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt (hunk 3 above)
Comment (ruiqima) [reply analyze-hotspots/src/main/java/com/esri/arcgismaps/sample/analyzehotspots/screens/MainScreen.kt:67]: Thanks for pointing this out. I will directly substitute the Viewpoint() here to simply the code.

**Pull request merged**